_authors/arnav-mehra.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Arnav Mehra
+short_name: Arnav
+twitter:
```

Comment:
Please remove this line if its not applicable.

Otherwise this LGTM!

Reply:
Please also move this to the `_authors/osci_2023/` folder.

Reply:
Thanks for the feedback, Craig! I have made the requested changes.